tests/fixtures_mc_comm.py

```diff
@@ -19,3 +21,22 @@ def fixture_four_board_mc_comm_process():
         "error_queue": error_queue,
     }
     yield items_dict
+
+
+@pytest.fixture(scope="function", name="patch_comports")
+def fixture_patch_comports(mocker):
+    comport = "COM1"
+    mocked_comports = mocker.patch.object(
+        mc_comm.list_ports, "comports", autospec=True, return_value=[f"STM ({comport})"]
```

Comment:
would it be more robust to have this return a list of size larger than 1 (i.e. with maybe the first value not being a Mantarray but being some other COM device instead)? or do you think that adds minimal value to the robustness of the test suite?